multiprofit/psfmodel_utils.py

```diff
@@ -0,0 +1,105 @@
+# This file is part of multiprofit.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import gauss2d.fit as g2f
+
+import numpy as np
+from typing import List, Optional
+
+from multiprofit.limits import limits_ref
+
+
+def make_psf_source(
+    sigma_xs: Optional[List[float]] = None,
```

Comment:
For this (and other optional parameters in this PR) you can use `list[float] | None = None` for the type hint.